tests/phel/test/http.phel

```diff
@@ -230,7 +230,7 @@
   (is
     (=
       "GET"
-      (get (h/request-from-globals (php-associative-array
+      (get (h/request-from-globals-args (php-associative-array
```

Comment:
After you applied the changes above the function has now five arguments that you need to provide. Therefore the test will look like this:
```
(deftest test-request-get
  (is
    (=
      "GET"
      (get 
        (h/request-from-globals-args
          (php-associative-array "REQUEST_METHOD" "GET")
          (php-associative-array)
          (php-associative-array)
          (php-associative-array)
          (php-associative-array)) 
        :method)
    )
    "request method get"))
```

The same applies for the other tests below that use `h/request-from-globals-args`

Reply:
I used & to avoid having empty associative arrays in tests that don't use those parameters since it seems cleaner. What do you think?